core/controllers/acl_decorators_test.py

```diff
@@ -1278,6 +1451,13 @@ def setUp(self):
             self.private_exp_id, self.owner_id)
         rights_manager.publish_exploration(self.owner, self.public_exp_id)
 
+    def test_cannot_publish_exploration_with_invalid_exp_id(self):
+        self.login(self.OWNER_EMAIL)
+        with self.swap(self, 'testapp', self.mock_testapp):
+            self.get_json(
+                '/mock/%s' % 'invalid_exp_id', expected_status_int=404)
```

Comment:
Here and elsewhere: when you're interpolating string constants, just inline them, e.g. '/mock/invalid_exp_id'.

Also, here and elsewhere, ideally change '/mock/' to be more descriptive as well, e.g. '/mock_publish_exploration/'. Otherwise it's quite hard to follow the tests.

Reply:
Done